crate_universe/src/config.rs

```diff
@@ -77,10 +77,26 @@ pub struct RenderConfig {
     /// The command to use for regenerating generated files.
     pub regen_command: String,
 
-    /// An optional configuration for rendirng content to be rendered into repositories.
+    /// An optional configuration for rendering content to be rendered into repositories.
     pub vendor_mode: Option<VendorMode>,
 }
 
+impl Default for RenderConfig {
```

Comment:
Why did this need to be explicitly implemented?

Reply:
I noticed some of the tests were explicitly deserializing a JSON literal instead of constructing a Rust literal and my interpretation is that that's because there is an unexpected behavioral difference between the "default" you get from Serde and from `Default::default()`.  So I figured it made sense to unify them by implementing `Default` to call the same default field generators that Serde does.  Probably worth me adding a comment above the `impl` if you're agree with the motivation behind the change.

Reply:
That sounds pretty good, yeah, I didn't know about this. A comment would be nice so someone doesn't try to delete it because they also didn't know 😄